modelci/experimental/model/model_structure.py

```diff
@@ -222,3 +252,36 @@ class Structure(BaseModel):
         default_factory=dict,
         example={'conv1': {'fc1': 'A'}}
     )
+
+    @classmethod
+    def from_model(cls, model): #noqa
```

Comment:
Your parser will encounter missing of layers such as 'ReLU', 'Dropout' that do not have weight parameters. There is a better implementation idea for this. Do not care about how we get the model structure from a model weight file now. 

Torchviz is good enough for us.